packages/codemods/src/lib/runTransform.ts

```diff
@@ -1,38 +1,28 @@
 /**
- * A simple wrapper around the jscodeshift CLI.
+ * A simple wrapper around the jscodeshift.
  *
  * @see jscodeshift CLI's usage {@link https://github.com/facebook/jscodeshift#usage-cli}
  * @see prisma/codemods {@link https://github.com/prisma/codemods/blob/main/utils/runner.ts}
  * @see react-codemod {@link https://github.com/reactjs/react-codemod/blob/master/bin/cli.js}
  */
-import path from 'path'
 
-import execa from 'execa'
+// @ts-expect-error We don't have this in types but need for workaround https://github.com/facebook/jscodeshift/issues/398
+import * as jscodeshift from 'jscodeshift/src/Runner'
 
-const getExecaArgs = () => {
-  if (process.platform === 'win32') {
-    return {
-      command: 'yarn jscodeshift',
-      cmdArgs: [],
-    }
-  } else {
-    /**
-     * We can't run jscodeshift with yarn (like `yarn jscodeshift -t ...`) on macos/linux
-     *
-     * @see {@link https://github.com/facebook/jscodeshift/issues/424}
-     *
-     * But on Windows, yarn jscodeshift does
-     */
-    const jscodeshiftExecutable = path.resolve(
-      __dirname,
-      '../../../../node_modules/.bin/jscodeshift'
-    )
-
-    return {
-      command: 'node',
-      cmdArgs: [jscodeshiftExecutable],
-    }
-  }
+const defaultJscodeshiftOpts = {
+  verbose: 0,
+  dry: false,
+  print: false,
+  babel: true,
+  extensions: 'js',
+  ignorePattern: '',
+  ignoreConfig: [],
+  runInBand: false,
+  silent: false,
+  parser: 'babel',
+  parserConfig: {},
+  failOnError: false,
```

Comment:
I think perhaps we should set failOnError to be true. Not sure how to reproduce though to test